vault/changelog.release.2022-08-09.md

```diff
@@ -0,0 +1,71 @@
+
+## Highlights
+
+- feat(view): add "Toggle PreviewLock" command (#3293) @sam ([[docs|dendron.topic.preview.commands.toggle-preview-lock]])
+
+## Everything Else
+
+- enhance(structure): support xvault template in note traits (#3329) @joshi ([[docs|dendron://dendron.dendron-site/dendron.topic.traits.api#return]])
+- enhance(publish): speed up published site (#3336) @hunter
+- enhance(views): toggle preview lock from inside preview (#3327) @sam ([[docs|dendron.topic.preview.commands.toggle-preview-lock]])
+- enhance(workspace): only show the feature showcase once @kevin
+- enhance(lookup): add space between note title and vault name (#3347)[Sean Li](https://github.com/saifahn)
+- fix(views): nested bullets formatted correctly in preview (#3326) [cquick01](https://github.com/cquick01)
+- fix(lookup): validate file name on note lookup (#3312) @hikchoi ([[docs|dendron.topic.lookup.ref.restrictions]])
+- fix(workspace): remote vault not recognized by dendron for windows (#3316) @joshi
+- fix(workspace): always treat `begin|end` block anchors as valid (#3339) @hikchoi
+- fix(views): resolve issues with preview lock button (#3353) @sam
+
+## Community
+
+### General Updates
+
+### TIL Highlights
+
+> These are highlights from the [[Dendron Discord|dendron://dendron.dendron-site/community.discord.channels]] `#today-i-learned` channel.
+
+- 💡 `aleksey#5276` shared his learning that you can generate a battery report in windows by running powercfg /batteryreport /output "C:\battery_report.html" in the admin CMD.
+
+- 💡 `icanteven#0264` shared a link to [[Markdown] An option to highlight a "Note" and "Warning" using blockquote (Beta)](https://github.com/community/community/discussions/16925)
+
+
+### Dendron Reading Series
+
+This week's entry in the [[Dendron Reading Series|dendron://dendron.dendron-site/community.events.reading-series]].
+
+### Event Reminders
+
+- **Greenhouse Talks:** Visit the [[Greenhouse Talks|dendron://dendron.dendron-site/community.events.greenhouse]] for notes from previous sessions.
+    - Next: [Fri, Aug 26, 09:00 AM PST / 00:00 UTC](https://link.dendron.so/luma)
+    - [Greenhouse Talk Recordings - YouTube Playlist](https://link.dendron.so/greenhouse)
+- **Office Hours:** Visit the [[Office Hours page|dendron://dendron.dendron-site/community.events.office-hours]] for notes from previous sessions.
+    - Next: [Wed, Sep 07, 09:00 AM PST / 17:00 UTC](https://link.dendron.so/luma)
+    - [Office Hour Recordings - YouTube Playlist](https://link.dendron.so/6yPa)
+- **Community Teatime:** Drink tea, eat food, and hangout with the Dendron crew!
+    - Next: [Thu, Aug 18, 11:00 AM PST / 19:00 UTC](https://link.dendron.so/luma)
+- **New User Tuesdays:** Visit the [[New User Tuesdays page|dendron://dendron.dendron-site/community.events.new-user-tuesdays]] for notes from previous sessions.
```

Comment:
cancelled this month. can we remove and also update the series to be on the third tuesday of the month?

Reply:
did not remove it in release notes but updated the date instead. Do you want to remove it completely? I also updated the date in the luma calendar.

Reply:
updating date is fine